README.md

```diff
@@ -242,6 +242,21 @@ These additional options are specific to pug templates and can be configured in
   Indent pug in template tags in single file components such as from vue or svelte.  
   _default_: `false`
 
+- `pugEmptyAttributes`  
+  Change behavior of boolean attributes.
+
+  Choices:
+
+  - `'as-is'` _default_ -> Nothing is changed.  
+    Example: `foo(a, b="", c)`
+  - `'none'` -> Every attribute with empty quotes will be replaced by a boolean version without quotes.  
```

Comment:
I'm not quite satisfied with the `replaced by a boolean version` part :thinking:  
It's a little bit confusing

Reply:
What about "Every attribute with empty quotes will have them removed"

Reply:
At least, a bit better

Applies also for https://github.com/prettier/plugin-pug/pull/142/files#diff-dd2c9e3ce59123522b95af9ce90edd8bceac03275e49d64680605524c75fd126R18